build/package/core.Dockerfile

```diff
@@ -1,29 +1,25 @@
 FROM golang:1.16 AS builder
 
-# Set necessary environmet variables needed for our image
-ENV GO111MODULE=on \
-    CGO_ENABLED=0 \
-    GOOS=linux \
-    GOARCH=amd64
-
-# Move to working directory /build
-WORKDIR /build
-
-# Copy and download dependency using go mod
-COPY go.mod .
-COPY go.sum .
+ARG git_tag
+ARG git_commit
+
+WORKDIR /go/src/github.com/nrc-no/core
+
+ENV GO111MODULE=on
+ADD go.mod .
+ADD go.sum .
+
 RUN go mod download
 
-# Copy the code into the container
-COPY . .
+ADD . .
 
-# Build the application
-RUN go build -o main .
+RUN go mod verify
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o main .
 
 # Build a small image
-FROM scratch
+FROM golang:alpine
```

Comment:
Why do we need golang here? Can we not run the binary with go?

Reply:
Yeah, good catch. Had some troubles with compiling in sqlite binaries that we use in tests